frontend/src/components/zaehlung/form/ZaehlungForm.vue

```diff
@@ -144,4 +146,53 @@ const contentHeight = computed(() => {
     daveUtils.cardactionHeight.value;
   return `${height}vh`;
 });
+
+function validateZaehlung(): void {
```

Comment:
Am besten für die drei Methoden für die Validierungslogik die Unittests analog zum Unittest im Backend schreiben.

Bei den Unittest kannst du dich am Datenportal orientieren. Da sind schon einige drinnen.

Reply:
Die beiden Methoden `validateKnotenLageForm` und `validateVerkehrForm` können direkt in die Methode `validateZaehlung` integriert werden, da diese an keiner anderen Stelle verwendet werden.